packages/lavapack/src/runtime-cjs-template.js

```diff
@@ -63,7 +74,7 @@
     const moduleObject = { exports: {} }
     const evalKit = {
       scopeTerminator: strictScopeTerminator,
-      globalThis,
```

Comment:
Is it possible that the existing implementation is missing to handle some cases where `globalThis` is nullish? Existing polyfill doesn't seem to address this. Here I'm changing both here and setting the global `globalThis` at :18

Reply:
cant change the key in `evalKit`, it needs to remain `globalThis`

Reply:
to answer your question, yes. this was a mistake so we should use the `globalRef` reference in the places where we intend to use the global obj